### ISSUE TITLE
IntelliCode not working for Python: Sorry, something went wrong activating IntelliCode support for Python.

### ISSUE THREAD

**Issue by TransparentLC** [MicrosoftDocs/intellicode#282]
**Product and Version** [VS/VSCode]: VSCode 1.55.0 (system setup, commit c185983a683d14c396952dd432459097bc7f757f, 2021-03-30T16:01:55.261Z)
**OS Version** [macOS/Windows]: Windows_NT x64 10.0.19041
**IntelliCode Extension Version**: 1.2.12
**Target Platform or Language** [e.g. Node.js]: Python

**Steps to Reproduce / Scenario:**

1. Open a .py file.
2. Wait until the toast appears: "Sorry, something went wrong activating IntelliCode support for Python. Please check the "Python" and "VS IntelliCode" output windows for details."

I'm using the latest stable version of VSCode (1.55.0), Python extension (2021.3.680753044), Pylance extension (2021.3.4), and VS IntelliCode extension (1.2.12). And I have installed the latest Python interpreter (3.9.3).

The Python code is highlighted as normal, but I can't use other editing features including type hints, code completion, code navigation... and debugging, or just run the code. Running any VSCode's Python command gives me an error toast "Command 'Python: \*\*\*' resulted in an error (command 'python.\*\*\*' not found)". No Python interpreter info displays in the bottom left of the VSCode window.

There is a huge output in IntelliCode's output window, and I can't find the Python extension's output window.

![](https://user-images.githubusercontent.com/47057319/113503023-efa70980-9561-11eb-8d6f-bd33efab8bfc.png)

The output: [errors.txt](https://github.com/MicrosoftDocs/intellicode/files/6254184/errors.txt)

I tried to reinstall the Python/Pylance/IntelliCode extension (after uninstall I removed these extensions' folders in `.vscode\extensions`) or change the Python language server to Pylance/Microsoft but doesn't work.

**Comment by DavidObando** [comment]
Hi @TransparentLC from the screenshot you posted it looks like the Python extension isn't activating correctly. Can you take a quick look into the "Log (Extension Host)" output for errors found during the Python extension activation, and share the errors here?

**Comment by TransparentLC** [comment]
[log-extension-host.txt](https://github.com/MicrosoftDocs/intellicode/files/6261273/log-extension-host.txt)

After checking the log I found Python is not the only extension that failed to load and throws a lot of JS code into the log. These errors are caused by ["vscode babel transform"](https://marketplace.visualstudio.com/items?itemName=AndyHsu.vscode-babel-transform) extension. I disabled (or uninstalled because the extension is no longer maintained) it and everything works now. 

Since it is not an issue of IntelliCode I should close this issue now.
